### DISCUSSION TITLE
Hotkeys

### DISCUSSION THREAD

**Discussion by unHAPPYlemon** [weihuoya/citra#704] [General]
Thank you for your work and for keeping this emulator alive. Please add hotkeys to the next update. I really want to swap between normal speed and alt/turbo-speed with a button on a controller. It would also be cool if you add on-screen hotkeable buttons. More power.

**Comment by Tappy-tech** [comment]
You can use fast forward like this by holding the button, that looks like a 3DS home button. How you didn't notice it when mapping controller?

**Reply by unHAPPYlemon** [reply]
I'm aware of this. But what I wanted was a toggle since I'm playing at 2x speed most of the time. There are just times where I needed to go 1x speed. Going through options every time is exhausting.
